packages/flex-plugin-e2e-tests/src/tests/step003.ts

```diff
@@ -4,7 +4,7 @@ import { spawn, logResult, assertion } from '../utils';
 
 // Install dependencies
 const testSuite: TestSuite = async ({ scenario }: TestParams): Promise<void> => {
-  const result = await spawn('npm', ['i'], { cwd: scenario.plugin.dir });
+  const result = await spawn('npm', ['i'], { cwd: scenario.plugin.dir, shell: true });
```

Comment:
If `shell` is to be true for _all_ cases, or at least for majority, then just add it as default inside the `spawn` command.